modules/deploy/pages/deployment-option/self-hosted/kubernetes/redpanda-cluster-recommendations.adoc

```diff
@@ -0,0 +1,91 @@
+
+== Set resource requests and limits for memory and CPU
+
+In a production cluster, the resources you allocate to Redpanda should be proportionate to your machine type. Redpanda Data recommends that you determine and set these values before deploying the cluster. For instructions on setting Pod resources, see xref:manage:kubernetes/manage-resources.adoc[Manage Pod Resources in Kubernetes].
+
+== Use local PersistentVolumes backed by NVMe disks
+
+Redpanda Data recommends using local PersistentVolumes (PVs) that are backed by NVMe devices to store the Redpanda data directory. NVMe devices outperform traditional SSDs or HDDs.
+
+Redpanda Data recommends creating StorageClasses that use the https://github.com/metal-stack/csi-driver-lvm[local volume manager (LVM) CSI driver] to automatically provision PVs. The LVM allows you to group physical storage devices into a logical volume group. Allocating logical volumes from a logical volume group provides greater flexibility in terms of storage expansion and management. The LVM supports features such as resizing, snapshots, and striping, which are not available with the other drivers such as the local volume static provisioner.
+
+By default, the Redpanda Helm chart uses the default StorageClass in your Kubernetes cluster to create one PersistentVolumeClaim (PVC) for each Redpanda broker. To learn how to configure a different StorageClass, see xref:manage:kubernetes/storage/configure-persistent-storage.adoc[].
+
+For details about StorageClasses, see the https://kubernetes.io/docs/concepts/storage/storage-classes/[Kubernetes documentation^].
+
+== Use a NodePort Service for external access
+
+The NodePort Service provides the lowest latency of all the Kubernetes Services because it does not include any unnecessary routing or middleware. Client connections go to the Redpanda brokers in the most direct way possible, through the worker nodes.
+
+By default, the Redpanda Helm chart creates a NodePort Service with the following ports:
+
+|===
+| Listener|Node Port
+
+| Schema registry
+| 30081
+
+| HTTP Proxy
+| 30082
+
+| Kafka API
+| 31092
+
+| Admin API
+| 31644
+|===
+
+To change these ports, see xref:manage:kubernetes/networking/configure-listeners.adoc[].
+
+Depending on your deployment and security policies, you may not be able to access worker nodes through a NodePort Service.
+If you choose to use another Service, consider the impact on the cost and performance of your deployment:
+
+* *LoadBalancer Service*: To make each Redpanda broker accessible with LoadBalancer Services, you need one LoadBalancer Service for each Redpanda broker so that requests can be routed to specific brokers rather than balancing requests across all brokers. Load balancers are expensive, add latency and occasional packet loss, and add an unnecessary layer of complexity.
```

Comment:
```suggestion
* *LoadBalancer Service*: To make each Redpanda broker accessible with LoadBalancer Services, you need one LoadBalancer Service for each Redpanda broker. With this, requests can be routed to specific brokers, rather than balancing requests across all brokers. Load balancers are expensive, add latency and occasional packet loss, and add an unnecessary layer of complexity.
```